### ISSUE TITLE
from torch.optim.lr_scheduler import _LRScheduler 报错

### ISSUE THREAD

**Issue by Spacei567** [XPixelGroup/BasicSR#596]
在lr_scheduler.py文件中 
![屏幕截图 2023-02-19 220509](https://user-images.githubusercontent.com/74165106/219953288-dc2d8c37-a9c2-4c64-954d-c7effa4c9e3e.png)
请问怎么解决？

**Comment by Reality-Dream7** [comment]
您好，我也遇到这个问题了，请问您解决了吗？

**Comment by wwhio** [comment]
`_LRScheduler` 是 Pytorch 提供的各种 LR Scheduler 的基类，请检查：
1. 是否在 编辑器 / IDE 中选择了正确的 Python 解释器
2. 是否正确安装了 pytorch 

参考：
https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html
